### ISSUE TITLE
Add gem "conservative" option

### ISSUE THREAD

**Issue by nervo** [ansible/ansible#18925]
_From @nervo on December 7, 2014 20:57_

##### Issue Type:

Feature Idea
##### Ansible Version:

All known stable version
##### Environment:

N/A
##### Summary:

Adding a gem "conservative" option should avoid gem module to be downloaded and installed everytime the module is played.

See gem option help:

```
    --conservative               Don't attempt to upgrade gems already
                                 meeting version requirement
```
##### Steps To Reproduce:

Define a task:

```
- name: packages
  gem: >
    name=sass
    version=>0
    state=present
    user_install=no
  sudo: yes
```
##### Expected Results:

The gem module should be installed only first time the task is runned
##### Actual Results:

The gem module is installed every time the task is runned

(Hello ansibot :))

_Copied from original issue: ansible/ansible-modules-core#461_

**Comment by nervo** [comment]
_From @mpdehaan on December 8, 2014 22:19_

Typically we do this as force=no (default should probably be no).   Maybe we just want to fix this altogether, but upgrade behavior might not be clear.

**Comment by nervo** [comment]
_From @hyvs on May 27, 2015 12:58_

Any news about this feature ?

**Comment by nervo** [comment]
_From @bcoca on May 27, 2015 13:26_

I have no info of anyone working on it yet

**Comment by nervo** [comment]
ping @mpdehaan ?

**Comment by nervo** [comment]
_From @ansibot on August 1, 2016 3:47_

@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by nervo** [comment]
_From @ansibot on September 13, 2016 2:0_

@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by nervo** [comment]
_From @ansibot on October 6, 2016 16:44_

@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by nervo** [comment]
_From @ansibot on December 7, 2016 18:9_

This repository has been locked. All new issues and pull requests should be filed in https://github.com/ansible/ansible

Please read through the <a href="http://docs.ansible.com/ansible/dev_guide/repomerge.html#move-issues-and-prs-to-new-repo">repomerge</a> page in the dev guide. The guide contains links to tools which automatically move your issue or pull request to the ansible/ansible repo.

**Comment by ansibot** [comment]
@nervo Greetings! Thanks for taking the time to open this issue. In order for the community to handle your issue effectively, we need a bit more information. 

Here are the items we could not find in your description:
- component name

Please set the description of this issue with this template:
https://raw.githubusercontent.com/ansible/ansible/devel/.github/ISSUE_TEMPLATE.md

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibotdev** [comment]
@nervo You have not responded to information requests in this issue so we will assume it no longer affects you. If you are still interested in this, please create a new issue with the requested information.

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)
